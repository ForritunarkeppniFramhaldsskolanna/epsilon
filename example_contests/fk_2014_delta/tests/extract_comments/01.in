#include <iostream> //na i bokasofn
using namespace std;//nota tad
//main fallid
void main() {
    //skrifa ut tolu
    printf("%d\n", 10);
}
