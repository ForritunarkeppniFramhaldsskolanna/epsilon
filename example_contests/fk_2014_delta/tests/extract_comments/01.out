na i bokasofn
nota tad
main fallid
skrifa ut tolu
